modules/components/pages/inputs/redpanda_migrator.adoc

```diff
@@ -184,6 +187,31 @@ An optional consumer group. When specified, the partitions of specified topics a
 
 *Type*: `string`
 
+=== `output_resource`
+
+The label of the `redpanda_migrator` output in which the currently selected topics need to be created before attempting to read messages.
```

Comment:
Maybe add a link to the output docs and clarify that this is a separate connector.

Reply:
Updated as requested on both the input and output of this component.